cloud/deployment/fromfile/fromfile.go

```diff
@@ -592,27 +770,14 @@ func hasAlertEmails(deploymentFromFile *inspect.FormattedDeployment) bool {
 	return len(deploymentFromFile.Deployment.AlertEmails) > 0
 }
 
-// createAlertEmails takes a deploymentFromFile and deploymentID and a client as its arguments.
-// It creates or updates alert emails for the deployment identified by deploymentID.
-// It returns an error if it fails to update the alert emails for a deployment.
-func createAlertEmails(deploymentFromFile *inspect.FormattedDeployment, deploymentID string, client astro.Client) (astro.DeploymentAlerts, error) {
-	var (
-		alertsInput astro.UpdateDeploymentAlertsInput
-		alertEmails []string
-		alerts      astro.DeploymentAlerts
-		err         error
-	)
-
-	alertEmails = deploymentFromFile.Deployment.AlertEmails
-	alertsInput = astro.UpdateDeploymentAlertsInput{
-		DeploymentID: deploymentID,
-		AlertEmails:  alertEmails,
+func hasEnvVarsOrAlertEmails(deploymentFromFile *inspect.FormattedDeployment) bool {
+	if hasEnvVars(deploymentFromFile) {
+		return hasEnvVars(deploymentFromFile)
 	}
-	alerts, err = client.UpdateAlertEmails(alertsInput)
-	if err != nil {
-		return astro.DeploymentAlerts{}, err
+	if hasAlertEmails(deploymentFromFile) {
+		return hasAlertEmails(deploymentFromFile)
```

Comment:
I think we should simplify it to this
```suggestion
	return hasEnvVars(deploymentFromFile) || hasAlertEmails(deploymentFromFile)
```

Reply:
fixed

Reply:
simplified